packages/expo-sqlite/android/src/main/java/expo/modules/sqlite/SQLiteModule.kt

```diff
@@ -83,7 +83,9 @@ class SQLiteModule : Module() {
           if (bindArgs[i - 1] == null) {
             statement.bindNull(i)
           } else {
-            statement.bindString(i, bindArgs[i - 1])
+            bindArgs[i - 1]?.let {
+              statement.bindString(i, it)
+            } ?:  throw  IllegalArgumentException("the bind value at index $i is null");
```

Comment:
sorry to make confuse. i mean

```kotlin
          let arg = bindArgs[i - 1]
          if (arg == null) {
            statement.bindNull(i)
          } else {
            statement.bindString(i, arg)
          }
```

does it work for you?

Reply:
Ah yes, sorry, you want the smart cast. It's safe to use `!!` in this situation but your right, it's better to do it like this